components/src/dynamo/planner/utils/agg_planner.py

```
@@ -41,17 +41,17 @@ class AggPlanner:
+        self, runtime: Optional[DistributedRuntime], config: PlannerConfig
     ) -> None:
-        self.args = args
+        self.config = config
         self.shared_state = PlannerSharedState()
 
-        if getattr(args, "enable_throughput_scaling", False):
+        if config.enable_throughput_scaling:
             raise ValueError(
                 "Aggregated planner only supports load-based scaling. "
                 "Please use --disable-throughput-scaling or do not set --enable-throughput-scaling."
             )
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)